src/front-end/src/components/MentorForm.js

```diff
@@ -0,0 +1,126 @@
+              </Tooltip>
+            }
+          />
+        </Form.Item>
+        <Form.Item
+          label="Name"
+          name="nameInput"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input placeholder="please enter your name" />
+        </Form.Item>
+        <Form.Item
+          label="Email"
+          name="emailInput"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input placeholder="please enter your email" />
+        </Form.Item>
+        <Form.Item
+          label={
+            <span>
+              Timeframe Seeking Mentorship&nbsp;
+              <Tooltip title="Please pick an estimated duration for how long you would like to use the platform">
+                <QuestionCircleOutlined />
+              </Tooltip>
+            </span>
+          }
+          name="selectedStartDate"
+          rules={[{ required: true, message: 'This field is required' }]}
+        >
+          <RangePicker picker="month" />
+        </Form.Item>
+        <Form.Item
+          label="Please Select Your Time-zone"
+          name="timeZone"
+          // must have an input:
+          rules={[{ required: true, message: 'This field is required' }]}
+        >
+          <TimezonePicker
+            // time zones:
+            absolute={false}
+            defaultValue="America/Los_Angeles"
+            placeholder="Select timezone..."
+            onChange={this.handleChange}
+          />
+        </Form.Item>
+        <p> Please enter your reoccuring availability:</p>
+        <Form.Item label="Monday" name="availabilityMonday">
+          <Select
+            // so that multiple options can be selected:
+            mode="multiple"
+            style={{ width: '100%' }}
+            placeholder="select available time"
+            optionLabelProp="label"
+          >
+            <Option value="9-10AM" label="9-10AM" />
+            <Option value="10-11AM" label="10-11AM" />
+            <Option value="11AM-12PM" label="11AM-12AM" />
+            <Option value="12-1PM" label="12-1PM" />
+            <Option value="1-2PM" label="1-2PM" />
+          </Select>
+        </Form.Item>
+        <Form.Item label="Tuesday" name="availabilityTuesday">
+          <Select
+            // so that multiple options can be selected:
+            mode="multiple"
+            style={{ width: '100%' }}
+            placeholder="select available time"
+            optionLabelProp="label"
+          >
+            <Option value="9-10AM" label="9-10AM" />
+            <Option value="10-11AM" label="10-11AM" />
+            <Option value="11AM-12PM" label="11AM-12PM" />
+            <Option value="12-1PM" label="12-1PM" />
+            <Option value="1-2PM" label="1-2PM" />
+          </Select>
+        </Form.Item>
+        <Button type="primary" htmlType="submit">

```

Comment:
This section can be removed. 
Can you also add a TextArea tag to obtain a description about the Mentor? This should also be included on the Mentee form as well.
On just the Mentor form, can you add an input field to obtain a password from the user (and possibly add a ToolTip to explain that a password is required for registration to login to the mentorship portal)?

Reply:
TextArea is an input tag. I think you meant text tag?

Reply:
@wrc9817 An HTML input tag with the text attribute could work, however, this limits user input to one line, whereas TextAreas allow the user to type multiple lines (which, I would argue is more user friendly). 
Please let me know if my understanding of a 'text tag' is incorrect, and maybe that is what we'd need.

Reply:
I'm confused. I think you mean you want a description there right? I will use <text></text>.

Reply:
We need a [textarea](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea). If you look at the [ant design form page](https://ant.design/components/form/), they do it like this: 

```
<Form.Item name={['user', 'introduction']} label="Introduction">
        <Input.TextArea />
      </Form.Item
```

(Of course, change the specific labels of the field, but that's the idea).

Reply:
<img width="702" alt="Screen Shot 2020-11-13 at 1 33 56 PM" src="https://user-images.githubusercontent.com/21204917/99072745-e8805e80-25b4-11eb-9b75-4f53d34ef98d.png">

textarea